### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by OR13**
- [x] Add CBOR-LD / BBS QR Code Example
- [x] Fix controller
- [x] Update cached did doc example

### CODE REVIEW

**Review by mprorock — APPROVED** [review]

**Review by kimdhamilton — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -614,6 +633,60 @@ <h3 id="Credential"><dfn>Credential</dfn></h3>
...
+      "familyName": "SMITH",
+      "gender": "Male",
+      "birthDate": "1958-07-17"
+    },
+    "vaccine": {
+      "type": "Vaccine",
+      "disease": "COVID-19",
+      "atcCode": "J07BX03",
+      "medicinalProductName": "COVID-19 Vaccine Moderna",
+      "marketingAuthorizationHolder": "Moderna Biotech"
```
Comment (TallTed) [inline docs/index.html:671]: I think these would be better as made-up values, without reference to any real entity.
```suggestion
      "disease": "Example Virus",
      "atcCode": "J07BX03",
      "medicinalProductName": "Example Vaccine Example Biotech",
      "marketingAuthorizationHolder": "Example Biotech"
```

File: docs/index.html [hunk 2]
```diff
@@ -614,6 +633,60 @@ <h3 id="Credential"><dfn>Credential</dfn></h3>
...
+    "https://w3id.org/vaccination/v1",
+    "https://w3id.org/security/bbs/v1"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "VaccinationCertificate"
+  ],
+  "id": "urn:uvci:af5vshde843jf831j128fj",
+  "name": "COVID-19 Vaccination Certificate",
+  "description": "COVID-19 Vaccination Certificate",
```
Comment (TallTed) [inline docs/index.html:649]:
```suggestion
  "name": "Example Virus Vaccination Certificate",
  "description": "Example Virus Vaccination Certificate",
```

File: docs/examples/vaccination-event-credential-example.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      "familyName": "SMITH",
+      "gender": "Male",
+      "birthDate": "1958-07-17"
+    },
+    "vaccine": {
+      "type": "Vaccine",
+      "disease": "COVID-19",
+      "atcCode": "J07BX03",
+      "medicinalProductName": "COVID-19 Vaccine Moderna",
+      "marketingAuthorizationHolder": "Moderna Biotech"
```
Comment (TallTed) [inline docs/examples/vaccination-event-credential-example.json:35]:
```suggestion
      "disease": "Example Virus",
      "atcCode": "J07BX03",
      "medicinalProductName": "Example Virus Vaccine Example Biotech",
      "marketingAuthorizationHolder": "Example Biotech"
```

File: docs/examples/vaccination-event-credential-example.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    "https://w3id.org/vaccination/v1",
+    "https://w3id.org/security/bbs/v1"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "VaccinationCertificate"
+  ],
+  "id": "urn:uvci:af5vshde843jf831j128fj",
+  "name": "COVID-19 Vaccination Certificate",
+  "description": "COVID-19 Vaccination Certificate",
```
Comment (TallTed) [inline docs/examples/vaccination-event-credential-example.json:13]:
```suggestion
  "name": "Example Virus Vaccination Certificate",
  "description": "Example Virus Vaccination Certificate",
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by OR13** [comment]
@TallTed this example is pulled from https://github.com/w3c-ccg/vaccination-vocab and your changes would break the signature.

**Comment by TallTed** [comment]
*sighs* I guess I'll poke there now...

**Pull request merged**